06-Customizing AzTS for your org/README.md

```diff
@@ -3,13 +3,22 @@
 
 # Extending AzTS
 
-## Add new control for existing SVT
+## Customizing AzTS for your org
 
-Azure Tenant Security Solution (AzTS) provides capability to add new controls(for existing services supported by AzTS) to customize the AzTS for your organization as per your need.
-Currently you can extend AzTS controls set by either adding a new control based on custom Azure Policy or based on Microsoft Defender for Cloud assessment.
+### Control Metadata Editor tool
+CMET (Control Metadata Editor Tool) is a tool that can be used by privileged users (mostly by admins maintaining AzTS setup for organization)to customize AzTS for organization. It provides capabilities to update control metadata (Tags, Display Name, control settings(if applicable)), enable or disable control, etc.
```

Comment:
Pls. update as per the earlier recommendation. 
Same for other places where the common text/paragraph has been used.

Reply:
Updated